pcdsdevices/tests/test_utils.py

```diff
@@ -26,7 +26,7 @@
 @pytest.fixture(scope='function')
 def sim_input(monkeypatch):
     master, slave = pty.openpty()
-    with open(slave, 'r') as fake_stdin:
+    with open(slave) as fake_stdin:
```

Comment:
should I be omitting the mode when it's `'r'` from now on?

Reply:
It's the default (`open(..., mode='r')` so it's at least technically unnecessary

Reply:
Resolving this to clear the PR comments; we discussed this at length